### ISSUE TITLE
Publication of model parameters 

### ISSUE THREAD

**Issue by fratajcz** [snap-stanford/GEARS#50]
Hi!

Thanks again for making this code public. I have found the hyperparameter settings that you have used to train your model in the supplementary notes of the paper. However, I could not find a link to download the parameters of the trained model which has produced the performance metrics and predictions listed in the paper. I am asking since my training run on Norman et al. data looks fishy, i.e. loss decreases only minimally and then starts oscillating, and using the optimal parameters from the manuscript would greatly increase the confidence in my results and the method. 

Since it is good scientific practice to publish the model parameters, is there any chance we can download them? Perhaps there is a link somewhere and I have simply missed it, in that case I would be very greatful if you could post it here. 

Thank you

**Comment by fratajcz** [comment]
Hi! Thank you for your swift reply. I will test if my trained model roughly reproduces the figures and get back to you.
